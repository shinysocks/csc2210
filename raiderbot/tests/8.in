+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*TTT*TTT           ||
||                  ||
|+------------------+|
+--------------------+
19 9
N
WWWWWWNW
wWWWWWWWWWW
W
E
W

