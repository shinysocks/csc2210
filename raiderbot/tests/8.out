Initial state: Robot at 19, 9 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*TTT*TTT           ||
||                  ||
|+------------------R|
+--------------------+
========================================
Executing N
Robot at 19, 8 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*TTT*TTT           ||
||                  R|
|+------------------+|
+--------------------+
Executing WWWWWWNW
Robot at 12, 7 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*TTT*TTT    R      ||
||                  ||
|+------------------+|
+--------------------+
Executing wWWWWWWWWWW
Robot now owes the bank 1 pieces.
Robot now owes the bank 2 pieces.
Robot now owes the bank 3 pieces.
Robot now owes the bank 3 pieces.
Robot now owes the bank 4 pieces.
Robot now owes the bank 5 pieces.
Robot at 1, 7 (-5 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*RTT TTT           ||
||                  ||
|+------------------+|
+--------------------+
Executing W
Robot at 0, 7 (-4 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|RTTT TTT           ||
||                  ||
|+------------------+|
+--------------------+
Executing E
Robot now owes the bank 5 pieces.
Robot at 1, 7 (-5 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
| RTT TTT           ||
||                  ||
|+------------------+|
+--------------------+
Executing W
Robot at 0, 7 (-5 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|RTTT TTT           ||
||                  ||
|+------------------+|
+--------------------+
========================================
Robot has completed its task.
